### PR TITLE
Fix: draft behaviour and hydration error

### LINKED ISSUE #483: 🐛 Bug: Improve behaviour of Draft 2020-12 for mobile devices
### Describe the bug

The page with the release notes for draft 2020-12: https://json-schema.org/draft/2020-12/release-notes does not work well for mobile devices. The hold school tables producing a huge horizontal overflow. 

Screenshot 2024-03-08 at 18 34 38

### Steps To Reproduce

1. Go to https://json-schema.org/draft/2020-12/release-notes
2. Use developer tools to navigate like a mobile 
3. Check the unnecessary horizontal overflow 

### Expected Behavior

The ideal solution for me is a grid that when it reaches the breaking point converts that 1 row x 2 columns approach into a 2 rows x 1 column so we avoid the overflow to happen. 

### Screenshots

_No response_

### Device Information [optional]

```markdown
- OS:
- Browser:
- version:
```

### Are you working on this issue?

No

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No

### PR DESCRIPTION
**Pull request by AdityaSingh-02**
**What kind of change does this PR introduce?**

**Issue Number:**

-  Closes #483  #487  
-  Related to #___ 
-  Others? 

**Screenshots/videos:**

**Before**

https://github.com/json-schema-org/website/assets/94185167/5a59b99d-f71f-4855-814e-6cbf271dff91

**After**

https://github.com/json-schema-org/website/assets/94185167/9a33a0ff-bf5a-471f-aeb6-df5fd49d8c7a

**If relevant, did you update the documentation?**

**Summary**
1- Here I have fixed the behavior of the table before it was taking full width and there was not any overflow or width property.
2- I have also made changes in the release-notes.md, which was causing a hydration error and  I made a change at line number 466. it was taking more than required width.

**Does this PR introduce a breaking change?**
- No

### CODE REVIEW

**Review by DhairyaMajmudar — COMMENTED** [review]
Can you pls. remove the changes from yarn.lock and package.lock.json file since it is increasing the pr size.

**Comment by DhairyaMajmudar** [comment]
Also it will be good for users, if instead of applying overflow-auto , the 1x2 grid is converted into 2x1 grid as suggested in the issue : )

**Review by JeelRajodiya — CHANGES_REQUESTED** [review]
I think you deleted the entire yarn.lock file. 
As @DhairyaMajmudar mentioned, please leave the yarn.lock file as it is ( do not make any changes nor delete it  )

**Comment by AdityaSingh-02** [comment]
> I think you deleted the entire yarn.lock file. As @DhairyaMajmudar mentioned, please leave the yarn.lock file as it is ( do not make any changes nor delete it )

Yup, I deleted it completely(by mistake), I should have stashed changes before pushing, I will take care of it from next time, Till then what should I do? will it affect the code? since I have not added any new dependencies, the file has minor changes it should not affect code base(I might be wrong here)

**Comment by aialok** [comment]
@AdityaSingh-02, a better approach would be to clone the project from the beginning using Yarn and leave this as it is. Check out the `README.md` for more details. 
Then, create a new branch with the same name as the one you used in this PR and make the required changes or you can copy-paste your work and force-push the changes to this PR branch. This will solve all your problems. Before cloning the project again, make sure you have fetched the latest upstream in your fork repository to avoid any conflict.

**Comment by AdityaSingh-02** [comment]
Heyy @JeelRajodiya I have reverted the changes, could you look at it again?

**Comment by JeelRajodiya** [comment]
@AdityaSingh-02, are you using npm ?

**Comment by AdityaSingh-02** [comment]
> @AdityaSingh-02, are you using npm ?

yup

**Comment by JeelRajodiya** [comment]
> > @AdityaSingh-02, are you using npm ?
> 
> yup

yes, that is why the yarn.lock file is being modified. 

You'll have **use yarn** instead of npm as suggested by @aialok in [this](https://github.com/json-schema-org/website/pull/498#issuecomment-1987263107) comment

**Comment by AdityaSingh-02** [comment]
> > > @AdityaSingh-02, are you using npm ?
> > 
> > 
> > yup
> 
> yes, that is why the yarn.lock file is being modified.
> 
> You'll have **use yarn** instead of npm as suggested by @aialok in [this](https://github.com/json-schema-org/website/pull/498#issuecomment-1987263107) comment

okay got it, so should I do everything again or this works?

**Comment by JeelRajodiya** [comment]
> > > > @AdityaSingh-02, are you using npm ?
> > > 
> > > 
> > > yup
> > 
> > 
> > yes, that is why the yarn.lock file is being modified.
> > You'll have **use yarn** instead of npm as suggested by @aialok in [this](https://github.com/json-schema-org/website/pull/498#issuecomment-1987263107) comment
> 
> okay got it, so should I do everything again or this works?

can you run `yarn install` command and let me know if the yarn.lock file changes or not?

**Comment by AdityaSingh-02** [comment]
Heyy @aialok @JeelRajodiya Thank you for your guidance, I followed your steps and pushed changes using yarn, and this time yarn.lock file didn't change

**Review by JeelRajodiya — APPROVED** [review]

**Comment by JeelRajodiya** [comment]
Hey @AdityaSingh-02, You have resolved the problem. 
but I want to bring this section of the #483 to your attention. 
image


Let's just wait for the maintainer's review

**Comment by benjagm** [comment]
It is not the solution I proposed but I like it. Thanks!

**Review by benjagm — APPROVED** [review]

**Pull request merged**
